src/store/getters/state-getters.ts

```diff
@@ -468,6 +468,7 @@ export const isDissolutionValid = (state: StateIF): boolean => {
   const isCourtOrderValid = isRoleStaff(state) ? getCourtOrderStep(state).valid : true
   const isStaffPaymentValid = isRoleStaff(state) ? getStaffPaymentStep(state).valid : true
 
+  // eslint-disable-next-line
```

Comment:
Why not split the line instead?

Also, you should disable all lint for a line; only the error you want to disable.

Also, Megan is updating this anyway: https://github.com/bcgov/business-create-ui/pull/422#discussion_r917089556

Reply:
^^ What I would do is make this multi-line in this PR, and then if Megan's PR goes first then you can remove it. If Megan's PR goes second, she can remove it.